### DISCUSSION TITLE
[StreamPark] FileUtils.getPathFromEnv: HADOOP_HOME is not set on system env

### DISCUSSION THREAD

**Discussion by green241** [apache/streampark#2102] [General]
**- Question desc :**
When start a flink-sql job , streampark platform appears a problem ,as following : 
![image](https://user-images.githubusercontent.com/14801784/204081303-ca9b4682-0e9e-4a69-af94-c4d752612814.png)

**backend logs ,as following :** 
![image](https://user-images.githubusercontent.com/14801784/204080821-5012a951-9a31-4d3d-bb15-19de1ec1a0df.png)

**- Environment info :** 
flink :  1.14.3-2.12
k8s  : v1.20.9
Note :  There is no hadoop  installed in this test env !

**- Execute mode :** 
  k8s-application

**- Probably cause :** 
   When get flink job log achive path, the default file system is hdfs .

**Comment by wolfboys** [comment]
It's a bug, I'll fix it soon
